docs/en/observability/logs-stream.asciidoc

```diff
@@ -210,7 +210,7 @@ GET logs-example-default/_search
 }
 ----
 
-Your message, however, contains all of the following potential fields. Extracting these will allow you to filter and aggregate based on these fields:
+While you can search for phrases in the `message` field, you can't use it to filter log data. Your message, however, contains all of the following potential fields you can extract and use to filter and aggregate your log data:
```

Comment:
I'm not sure what "it" is referring to in this sentence. I can't use the search to filter log data?